completions/valgrind

```diff
@@ -84,8 +84,7 @@ _comp_cmd_valgrind()
                     return
                     ;;
                 \<+([0-9])..+([0-9])\>)
-                    COMPREPLY=($(compgen -W "{${value:1:${#value}-2}}" \
-                        -- "$cur"))
+                    _comp_compgen -- -W "{${value:1:${#value}-2}}"
```

Comment:
Maybe, per https://github.com/scop/bash-completion/pull/961#discussion_r1186844166
```suggestion
                    _comp_compgen -- -W '{${value:1:${#value}-2}}'
```

Reply:
I believe this intentionally makes the brace expansions actually take place.

(This is the second trial of the reply. I did reply to this comment, but my previous reply has disappeared. An incident that the notifications to PRs are not delivered is reported today ([GitHub Status - Incident with Pull Requests](https://www.githubstatus.com/incidents/wzv2wd9zvhpr)). It is not mentioned there, but I also observe that not only PRs but the notifications for Issues are not delivered in my project. It reminds me of the disappeared commits in https://github.com/scop/bash-completion/pull/955#discussion_r1186921818. Maybe the GitHub transactions are unstable this week.)